scripts/signature.py

```diff
@@ -114,6 +161,9 @@ def parse_args():
     diff_command_parser.add_argument("first", type=argparse.FileType("r"))
     diff_command_parser.add_argument("second", type=argparse.FileType("r"))
 
+    bias_command_parser = command.add_parser("bias")
+    bias_command_parser.add_argument("first", type=argparse.FileType("r"))
```

Comment:
nit: probably not in this diff, but using the "base/contender" terminology would make it consistent with what we use for benchmarks.